imports/plugins/core/cart/server/no-meteor/util/xformCartGroupToCommonOrder.js

```diff
@@ -1,4 +1,5 @@
 import { toFixed } from "accounting-js";
+import findVariantInCatalogProduct from "/imports/plugins/core/catalog/server/no-meteor/utils/findVariantInCatalogProduct";
```

Comment:
At some point recently I added `findVariantInCatalogProduct` as a context query. So you can remove this import and instead do `const { variant } = context.queries.findVariantInCatalogProduct(...)` below. That way we're not importing between plugins.

Reply:
Yes... I made a note in that PR to update this one when it was available, wasn't sure if it was yet. Will update.